Wavefront.CSharp.SDK.Test/UtilsTest.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using Wavefront.CSharp.SDK.Common;
+using Wavefront.CSharp.SDK.Entities.Histograms;
+using Xunit;
+
+namespace Wavefront.CSharp.SDK.Test
+{
+    /// <summary>
+    /// Unit tests for <see cref="Utils"/>.
+    /// </summary>
+    public class UtilsTest
+    {
+        [Fact]
+        public void TestSanitize()
+        {
+            Assert.Equal("\"hello\"", Utils.Sanitize("hello"));

```

Comment:
if you have a need to escape strings you can use string literals 

`@"""hello"""` though if it is more clear is up for debate.